docs/Set-PSReadLineKeyHandler.md

```diff
@@ -1,150 +1,157 @@
 ```
-Set-PSReadLineKeyHandler [-Chord] <String[]> [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>]
- [-Description <String>] [-ViMode <ViMode>]
+Set-PSReadLineKeyHandler [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>] [-Description <String>]
+ [-Chord] <String[]> [-ViMode <ViMode>] [<CommonParameters>]
 ```
 
-### Set 2
+### Function
 
 ```
-Set-PSReadLineKeyHandler [-Chord] <String[]> [-Function] <String> [-ViMode <ViMode>]
+Set-PSReadLineKeyHandler [-Chord] <String[]> [-ViMode <ViMode>] [-Function] <String> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-This cmdlet is used to customize what happens when a particular key or sequence of keys is pressed while PSReadLine is reading input.
-
-With user defined key bindings, you can do nearly anything that is possible from a PowerShell script.
-Typically you might just edit the command line in some novel way, but because the handlers are just PowerShell scripts, you can do interesting things like change directories, launch programs, etc.
+The `Set-PSReadLineKeyHandler` cmdlet customizes the result when a key or sequence of keys is
+pressed. With user-defined key bindings, you can do almost anything that is possible from within a
+PowerShell script.
 
 ## EXAMPLES
 
-### --------------  Example 1  --------------
+### Example 1: Bind the arrow key to a function
 
-```
-PS C:\> Set-PSReadLineKeyHandler -Key UpArrow -Function HistorySearchBackward
+This command binds the up arrow key to the function **HistorySearchBackward**. This function uses
+the current contents of the command line as the search string used to search the command history.
+
+```powershell
+Set-PSReadLineKeyHandler -Chord UpArrow -Function HistorySearchBackward
 ```
 
-This command binds the up arrow key to the function HistorySearchBackward which will use the currently entered command line as the beginning of the search string when searching through history.
+### Example 2: Bind a key to a script block
 
-### --------------  Example 2  --------------
+This example shows how a single key can be used to run a command. The command binds the key
+`Ctrl+Shift+B` to a script block that clears the line, inserts the word "build", and then accepts
+the line.
 
-```
-PS C:\> Set-PSReadLineKeyHandler -Chord Shift+Ctrl+B -ScriptBlock {
->>    [Microsoft.PowerShell.PSConsoleReadLine]::RevertLine()
->>    [Microsoft.PowerShell.PSConsoleReadLine]::Insert('msbuild')
->>    [Microsoft.PowerShell.PSConsoleReadLine]::AcceptLine()
+```powershell
+Set-PSReadLineKeyHandler -Chord Ctrl+Shift+B -ScriptBlock {
+    [Microsoft.PowerShell.PSConsoleReadLine]::RevertLine()
+    [Microsoft.PowerShell.PSConsoleReadLine]::Insert('build')
+    [Microsoft.PowerShell.PSConsoleReadLine]::AcceptLine()
 }
 ```
 
-This example binds the key Ctrl+Shift+B to a script block that clears the line, inserts build, then accepts the line.
-This example shows how a single key can be used to execute a command.
-
 ## PARAMETERS
 
-### -Chord
+### -BriefDescription
```

Comment:
Why are the parameters reordered here? I won't merge this PR with reordered content like this - it's too hard to review and too easy to miss things, e.g. you've removed at least one parameter.